packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx

```diff
@@ -130,11 +135,12 @@ export const CreateDRPolicy: React.FC<ReRouteResourceProps> = ({
         payload: '',
       });
     }
-  }, [clustersData, t, dispatch]);
+  }, [state.selectedClusters, t, dispatch]);
 
   const onCreate = () => {
     const promises: Promise<K8sResourceKind>[] = [];
-    const peerNames = clustersData?.map((cluster) => cluster?.name) ?? [];
+    const peerNames =
+      state.selectedClusters?.map((cluster) => cluster?.name) ?? [];
```

Comment:
initial value of `state.selectedClusters = []`, so I think we should be safe, unless we are storing `null` or `undefined` anywhere to this state, are we ??
```suggestion
    const peerNames =
      state.selectedClusters.map((cluster) => cluster?.name);
```